http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -395,6 +414,11 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
     }
   }
 
+  private[service] def validateMinimumDiskSize(diskSize: DiskSize): F[Unit] = {
```

Comment:
we should just need to update https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala#L136...

Reply:
oh man lol i complicated this way more than i needed to. thanks for the headups, going to start a new branch and make this small change!